README.md

```diff
@@ -335,14 +335,14 @@ For more information on embedding inline screenshots and file downloads,
 ### Command Line Options
 
 Popular command line options are:
-- `--browsers ie8` or `--browsers ie8,chrome` - specify the browsers you wish to run against. This overrides the test level settings. Valid browsers can be found in your account settings.
+- `--platform ie8` or `--platform ie8,chrome` - specify the platforms you wish to run against. This overrides the test level settings. Valid platforms can be found in your account settings.
```

Comment:
```suggestion
- `--platforms ie8` or `--platforms ie8,chrome` - specify the platforms you wish to run against. This overrides the test level settings. Valid platforms can be found in your account settings.
```
Did we want to keep the plural or did we change to singular on purpose? Plural seems better given the help text.